doc/release/1.1.0-notes.rst

```diff
@@ -1,26 +1,28 @@
-==========================
+=========================
 SciPy 1.1.0 Release Notes
-==========================
+=========================
 
 .. note:: Scipy 1.1.0 is not released yet!
 
 .. contents::
 
-SciPy 1.1.0 is the culmination of X months of hard work. It contains
-many new features, numerous bug-fixes, improved test coverage and
-better documentation.  There have been a number of deprecations and
-API changes in this release, which are documented below.  All users
-are encouraged to upgrade to this release, as there are a large number
-of bug-fixes and optimizations.  Before upgrading, we recommend that users
-check that their own code does not use deprecated SciPy functionality
-(to do so, run your code with ``python -Wd`` and check for
-``DeprecationWarning`` s).  Our development attention will now shift to
-bug-fix releases on the 1.0.x branch, and on adding new features on the
-master branch.
 
+SciPy 1.1.0 is the culmination of 7 months of hard work. It contains
+many new features, numerous bug-fixes, improved test coverage and better
+documentation. There have been a number of deprecations and API changes
+in this release, which are documented below. All users are encouraged to
+upgrade to this release, as there are a large number of bug-fixes and
+optimizations. Before upgrading, we recommend that users check that
+their own code does not use deprecated SciPy functionality (to do so,
+run your code with ``python -Wd`` and check for ``DeprecationWarning``
+s). Our development attention will now shift to bug-fix releases on the
+1.1.x branch, and on adding new features on the master branch.
 
 This release requires Python 2.7 or 3.4+ and NumPy 1.8.2 or greater.
 
+This release has improved but not necessarily 100% compatibility with
+the `PyPy <https://pypy.org/>`__ Python implementation. For running on
+PyPy, PyPy 6.0+ and Numpy 1.15.0+ are required.
```

Comment:
Why PyPy 6.0? Version 6.0 hasn't been released yet.

Reply:
Because Pypy 5.x has bugs that make things to not work, and 6.0 will be released relatively soon iiuc.